homeassistant/components/group/__init__.py

```diff
@@ -300,6 +304,10 @@ async def reload_service_handler(service: ServiceCall) -> None:
             return
         await _async_process_config(hass, conf)
 
+        for e in auto:
+            # pylint: disable-next=protected-access
+            e._platform_state = EntityPlatformState.NOT_ADDED
+
```

Comment:
Can this be avoided in `group` ?

Reply:
Yes, it can be avoided and it would get rid of unnecessary state changes for groups created via service calls when reloading.
I think that should be in a separate PR though.